content/docs/guides/auth/ALTS_GCP.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ALTS on Google Cloud Platform (GCP)
+description: >
+  An overview of gRPC ALTS authentication running on Google Cloud Platform.
+---
+
+### Overview
+
+[ALTS](ALTS) is now available to all gRPC users, if the application runs on
+[Google Compute Engine (GCE)](https://cloud.google.com/compute/) or
+[Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine).
+The detailed instructions on how to use ALTS as an authentication mechanism
+in gRPC can be found in [ALTS Authentiction guide](ALTS).
+
+### Identity and Key Management
+
+Using ALTS transport security protocol on Google Cloud Platform, the identity
+of the gRPC application is the primary service account associated with the GCE
+VM that the application runs on. The service account of a GCE VM can be set or
+changed using
+[gCloud command](https://cloud.google.com/sdk/gcloud/reference/compute/instances/set-service-account)
+or via
+[GCP console](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#using).
```

Comment:
This is all very GCE centric. I assume the identity in Kubernetes is a given processes service account? It would be useful to clarify some of this in this guide.

Reply:
I added additional info about GKE. ALTS works for GKE, only using underlying node identity (GKE node is GCE VM). Per-pod identity is WIP.